core/src/main/java/org/elasticsearch/action/ingest/IngestActionFilter.java

```diff
@@ -101,6 +101,7 @@ void processIndexRequest(Task task, String action, ActionListener listener, Acti
     }
 
     void processBulkIndexRequest(Task task, BulkRequest original, String action, ActionFilterChain chain, ActionListener<BulkResponse> listener) {
+        long ingestStartTime = System.currentTimeMillis();
```

Comment:
The reason I picked `System.currentTimeMillis()` is because that is what the bulk api uses too. Otherwise that would be inconsistent?